scene/gui/graph_node.cpp

```diff
@@ -801,20 +801,26 @@ void GraphNode::_connpos_update() {
 
 		if (slot_info.has(idx)) {
 			if (slot_info[idx].enable_left) {
-				ConnCache cc;
-				cc.pos = Point2i(edgeofs, y + h / 2);
+				PortCache cc;
```

Comment:
Just a (very) small nitpick. Maybe too short so we could name it `port_cache`, but since it's only used in the following 5 lines it would be fine either way.
```suggestion
				PortCache pc;
```

Reply:
Yeah, I thought about it originally, but decided not to pollute the PR with an insignificant rename. It's local, indeed, so it's not too important. Can be replaced in a future refactoring.